### DISCUSSION TITLE
Q22FA12800332 substituted with CN4040M00012001

### DISCUSSION THREAD

**Discussion by VivantSenior** [justLV/onju-voice#63] [Q&A]
Is it possible to substitute [Q22FA12800332](https://www.lcsc.com/product-detail/Others_Seiko-Epson_Q22FA12800332_Seiko-Epson-Q22FA12800332_C255899.html) with [CN4040M00012001](https://item.szlcsc.com/8264504.html)?

**Comment by jhbruhn** [comment]
Looks fine to me, the specs match, if the websites you linked are correct.
